types.go

```diff
@@ -4,6 +4,9 @@ import (
 	"github.com/gogo/protobuf/types"
 )
 
+// NamespaceID defines the type of the namespace ID
+type NamespaceID [28]byte
```

Comment:
The namespace now contains two fields https://github.com/celestiaorg/celestia-app/blob/db15dcb2562d6f06ad698fb5ce84d655a7886e36/pkg/namespace/namespace.go#L8-L11

- version: 1 byte
- ID: 28 bytes

Given the usage of this type elsewhere in this PR I think it may make sense to rename to `Namespace` and bump to `29`. Alternatively, consider using the types exported by celestia-app

Reply:
Added import, thanks 🙏🏽